### PR TITLE
Allow external roles to be installed to galaxy_roles

### PR DESCRIPTION
**Pull request by akofink**
FYI: `ansible-galaxy` installs roles to the first role in your `roles_path`.

To test, try something like this:

```sh
ansible-galaxy install bennojoy.mysql
```

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@
 callback_plugins = $PWD/playbooks/callback_plugins/
 nocows = 1
 inventory = $PWD/playbooks/inventory/vagrant.py
-roles_path = $PWD/playbooks/roles
+roles_path = $PWD/galaxy_roles:$PWD/playbooks/roles
```
Comment (ehelms) [inline ansible.cfg]: I was thinking of putting this side-by-side for parity: `playbooks/galaxy_roles`

**Comment by ekohl** [comment]
Would it be useful if this was added to the readme?

**Comment by akofink** [comment]
I updated the Readme and also added a `requirements.yml.example` and ignored `requirements.yml` for use with `ansible-galaxy install -r requirements.yml`.

**Comment by ehelms** [comment]
Thanks @akofink

**Pull request merged**